source/iostreams.tex

```diff
@@ -5916,6 +5916,19 @@
 \postcondition
 \tcode{rdbuf() == sb}.
 
+\indexlibrary{\idxcode{basic_ostream}!constructor}%
```

Comment:
Looks like a mis-application of http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2005/n1862.html after the Oxford 2007 meeting that no-one had noticed yet!  I think the declaration in the class definition could be added editorially, given it was present in the original paper.